### ISSUE TITLE
the detail about BD Degradation

### ISSUE THREAD

**Issue by wenchen4321** [YapengTian/TDAN-VSR-CVPR-2020#22]
Can you please tell me the detail about BD degradation? What are the sigma and the kernel size of Gaussian kernel? Because the sigma of Gaussian kernel are different in Fsrvsr and DUF.

**Comment by YapengTian** [comment]
I used the BD resizing code from the RDN https://github.com/yulunzhang/RDN/blob/master/RDN_IR/RDN_TestCode/RDN_SR_RGB/Prepare_TestData_HR_LR.m.

**Comment by wenchen4321** [comment]
Thanks a lot! But I found the BD codes in rdn that the kernel size is odd number when scale factor is 3, and kernel size is even number when scale fator is 2,4,8. And I guess the kernel size of you BD_LR training dataset is that the sigma is 1.6 and kernel size is 12*12 if you follow the setting of RDN.  Have you train you model with the kernel size with 13*13?

**Comment by YapengTian** [comment]
Yes, I used the default setting as in RDN code. I did not try 13*13 during training. But I think 13*13 probably will be better since we usually use an odd value for the blurring kernel. Thanks for letting me learn it! You might try it when you generate training samples. In addition, I found that the models trained with gaussian blur kernels can get better results. There are several recent iccv and cvpr papers comparing with EDVR but using gaussian kernels which are unfair experimental comparison.
